client/ignore.lua

```diff
@@ -11,9 +11,9 @@ CreateThread(function()
 end)
 
 AddEventHandler('populationPedCreating', function(x, y, z)
-    Wait(500)    -- Give the entity some time to be created
-    local _, handle = GetClosestPed(vec3(x, y, z), {}) -- Get the entity handle
-    SetPedDropsWeaponsWhenDead(handle, false)
+	Wait(500)	-- Give the entity some time to be created
+	local handle = GetClosestPed(vec3(x, y, z)) -- Get the entity handle
+	SetPedDropsWeaponsWhenDead(handle, false)
```

Comment:
This looks indented too much?

Reply:
This one was done in the pr you just merged but I believe these are all 4 space indentations